docs/release-notes/4435-fix-default-user.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  Allow users to specify a default password for the admin and
+   determined accounts.
```

Comment:
Probably worth calling out this is specific to k8s deployments and that we are releasing a new version of the helm chart.